src/commands/circleci-stop-if.yml

```diff
@@ -0,0 +1,20 @@
+description: >-
+  Helper command which will `halt` the entire execution of the running `job` if the value of
+  tested parameter is the value of the expectation parameter.
+
+  when conditions are great, but you could use this as a pre-step in a workflow to avoid executing a job
+
+parameters:
+  tested:
```

Comment:
🤔 I almost want to use `actual` here to match unit testing convention - I know this isn't really in the same vein but this parameter name doesn't immediately fit in a known pattern to me. Is there a more standard term or is this a "me" problem?

Reply:
Hmmm, my latest change leans more into the English of it, because that I couldn't get over the "if a result matches an expectation in a unit test that's good, but if this result matches the expectation things will stop" mental roadblock.

I'm hoping
```
  circleci-stop-if:
     value: "foo"
     stop-if-value-is: "foo"
```

feels better?

Reply:
I like it!